bulk_sync_into_benchling/antibodies.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Did you consider making this a list at the top level?

Reply:
I wanted to make it similar to the existing `sync_into_benchling` example